django/contrib/gis/geos/point.py

```diff
@@ -9,6 +9,7 @@
 class Point(GEOSGeometry):
     _minlength = 2
     _maxlength = 3
```

Comment:
@timgraham 
Attributes are not ordered here, so I'm not sure how to proceed with your suggestion...